pytorch_to_returnn/torch/nn/modules/linear.py

```diff
@@ -123,6 +122,41 @@ def create_returnn_layer_dict(self, *inputs: Tensor, **kwargs) -> Dict[str, Any]
     return {"class": "dot", "red1": red1, "red2": red2, "var1": var1, "var2": var2, "from": sources}
 
 
+class DotProductAlongDim(Module):
+  """
+  Maps to RETURNN DotLayer for a special case
+  """
+  is_original_torch_module = False
+
+  def __init__(self, dim: int):
```

Comment:
I would rather call it `reduce_dim` or so.
But also, I wonder whether it is more useful to not have this argument in the `__init__` but directly in the `create_returnn_layer_dict`?